### PR TITLE
Created a react native file with login app page

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { StyleSheet, Text, View, Button } from 'react-native'
+
+export default function App() {
+    return (
+        <View style={styles.container}>
+            <Text style={styles.title}>GitHub Project Boards</Text>
+            <Button
```
Comment (Meharban-Singh) [inline App.js]: This button should have some kind of background color. Could you make it red?

**Review by Meharban-Singh — CHANGES_REQUESTED** [review]
Could you address this?

**Comment by ghost** [comment]
Unnecessary files were removed and login button colour was changed to red

File: App.js (hunk 1 above)
Comment (ghost) [reply App.js]: Changes have been made

**Pull request merged**